src/js_native_api_v8.h

```diff
@@ -411,30 +385,19 @@ class Reference : public RefBase {
   v8::Local<v8::Value> Get();
 
  protected:
-  void Finalize(bool is_env_teardown = false) override;
+  void Finalize() override;
 
  private:
+  static void WeakCallback(const v8::WeakCallbackInfo<Reference>& data);
+
   void ClearWeak();
   void SetWeak();
 
-  static void FinalizeCallback(
-      const v8::WeakCallbackInfo<SecondPassCallParameterRef>& data);
-  static void SecondPassCallback(
-      const v8::WeakCallbackInfo<SecondPassCallParameterRef>& data);
-
-  v8impl::Persistent<v8::Value> _persistent;
-  SecondPassCallParameterRef* _secondPassParameter;
-  bool _secondPassScheduled;
+  v8impl::Persistent<v8::Value> persistent_;
 
   FRIEND_TEST(JsNativeApiV8Test, Reference);
```

Comment:
Should we remove it because the test is removed?

Reply:
Yeah. Removed.